man/man8/zpool-attach.8

```diff
@@ -62,6 +69,38 @@ creates a three-way mirror, and so on.
 In either case,
 .Ar new_device
 begins to resilver immediately and any running scrub is cancelled.
+.Pp
+If the existing device is a RAIDZ device (e.g. specified as "raidz2-0"), the new
+device will become part of that RAIDZ group.
+A "raidz expansion" will be initiated, and the new device will contribute
+additional space to the RAIDZ group once the expansion completes.
+The expansion entails reading all allocated space from existing disks in the
+RAIDZ group, and rewriting it to the new disks in the RAIDZ group (including the
+newly added
+.Ar device ) .
+Its progress can be monitored with
+.Nm zpool Cm status .
+.Pp
+Data redundancy is maintained during and after the expansion.
+If a disk fails while the expansion is in progress, the expansion pauses until
+the health of the RAIDZ vdev is restored (e.g. by replacing the failed disk
+and waiting for reconstruction to complete).
+Expansion does not change the number of failures that can be tolerated
+without data loss (e.g. a RAIDZ2 is still a RAIDZ2 even after expansion).
+A RAIDZ vdev can be expanded multiple times.
+.Pp
+After the expansion completes, old blocks remain with their old data-to-parity
+ratio (e.g. 5-wide RAIDZ2, has 3 data to 2 parity), but distributed among the
```

Comment:
"remain with" -> "retain"; RAID-Z2; drop the comma before "has"

Also "3 data to 2 parity" reads weird without context; 3/2 maybe? how's about uhh
```
After the expansion completes, old blocks retain their old data/parity ratio
(e.g. 5-wide RAID-Z2 is 3/2), but distributed among the
```
or maybe
```
After the expansion completes, old blocks retain their old data/parity ratio
.Pq e.g. Sy 5 Ns -wide RAID-Z2 is Sy 3 Ns / Sy 2 ,
but distributed among the
```
?

Reply:
How about "3 data and 2 parity".  I think "3/2" is confusing because it doesn't make sense read as a fraction.
```
After the expansion completes, old blocks retain their old data-to-parity
ratio 
.Pq e.g. 5-wide RAID-Z2 has 3 data and 2 parity
but distributed among the larger set of disks.
```

Reply:
Looks good